testdata/src/main/java/com/hedvig/app/testdata/feature/insurance/builders/InsuranceContractBuilder.kt

```diff
@@ -21,7 +21,7 @@ class InsuranceContractBuilder(
     private val detailsTable: TableFragment = TableFragmentBuilder().build(),
 ) {
 
-    fun build() = InsuranceQuery.Contract(
+    fun build(): InsuranceQuery.Contract = InsuranceQuery.Contract(
```

Comment:
```suggestion
    fun build() = InsuranceQuery.Contract(
```

This type is implicit from the functional syntax, and can therefore be omitted for the sake of readability 🙂

Reply:
It got changed by default by the IDE when I extracted a local variable which made this function to not have this functional syntax. Changed it back but forgot to remove the explicit return type. Fixing it now.